### PR TITLE
ports/mimxrt/boards/MIMXRT1060_EVK: Enabled led class for MIMXRT1060_EVK

### PR DESCRIPTION
**Pull request by Albort12138**
board.

- Enabled led class for MIMXRT1060_EVK.

### CODE REVIEW

File: ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.h [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 #define BOARD_FLASH_SIZE (8 * 1024 * 1024)
 
-#define MICROPY_HW_LED_PINMUX  IOMUXC_GPIO_AD_B0_09_GPIO1_IO09
-#define MICROPY_HW_LED_PORT    GPIO1
-#define MICROPY_HW_LED_PIN     9
+// MIMXRT1060_EVK has 1 user LED
+#define MICROPY_HW_LED1_PIN (GPIO_AD_B0_09)
+#define MICROPY_HW_LED_ON(pin) (mp_hal_pin_high(pin))
```
Comment (alphaFred) [inline ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.h:8]: According to my understanding of the schematics the user LED is pulled up to 3.3V supply. To switch it on pin should be pulled low I believe.

File: ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.h [hunk 2]
```diff
@@ -3,6 +3,7 @@
...
+#define MICROPY_HW_LED_OFF(pin) (mp_hal_pin_low(pin))
```
Comment (alphaFred) [inline ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.h:9]: Respectively here. To switch it off, pin should be pulled high.

**Review by alphaFred — CHANGES_REQUESTED** [review]
Hi, thanks for creating the change. I believe you need to make a small adjustment to the macros for turning the LED on/off, if my understanding of the schematics is correct. 

Best regards
alphaFred

**Comment by yanyao12138** [comment]
> Hi, thanks for creating the change. I believe you need to make a small adjustment to the macros for turning the LED on/off, if my understanding of the schematics is correct.
> 
> Best regards
> alphaFred

Yes, you are right, that's my fault, I have updated the PR.

**Comment by alphaFred** [comment]
> Yes, you are right, that's my fault, I have updated the PR.

No problem. I made the same mistake with my PR for adding support to 1020EVK 😄.

**Comment by yanyao12138** [comment]
> > Yes, you are right, that's my fault, I have updated the PR.
> 
> No problem. I made the same mistake with my PR for adding support to 1020EVK 😄.

Thank you very much! So you are now supporting 1020EVK? If it is OK, can you please share your plan for the support of the mimxrt port. So that we can avoid doing the duplicate work😊.

**Comment by alphaFred** [comment]
> So you are now supporting 1020EVK?
Well that is the board I have sitting on my desk hence I am eager to add it to the `boards` 😄. 

As you know I started working on the `pin` module. So looking forward to exchanging ideas with you about that. I don’t know if this is easier via an issue, PR or email etc.
Once that is done I would like to work on `SPI` module.

**Comment by dpgeorge** [comment]
Thanks for this!  Rebased and merged in e0d539f79d4ec020636ba7b043fac08030234112

**Pull request closed without merging**